web/web.go

```diff
@@ -67,6 +67,17 @@ import (
 
 var localhostRepresentations = []string{"127.0.0.1", "localhost"}
 
+// SecureHeaders adds secure headers to the responses
+func secureHeadersMiddleware(h http.Handler) http.Handler {
```

Comment:
since this is not exported and quite obvious what it does I don't think we need a comment here.

Reply:
All functions should have comments.

Reply:
why is that?

Reply:
The Go style guide.

Reply:
I wonder what is the reason for that.

All the info I could find is only for exported functions and non trivial unexported ones.
https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

> All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations. See https://golang.org/doc/effective_go.html#commentary for more information about commentary conventions.

IIUC `secureHeadersMiddleware` is trivial so should not need a comment and don't see how  would it be useful.